### ISSUE TITLE
Apps installed through TrollStore can't get files through iTunes File Sharing desktop app

### ISSUE THREAD

**Issue by mozbius** [opa334/TrollStore#365]
I have noticed that both my iPhone 7 and X can't receive files through iTunes desktop app on two different desktops. Only the hourglass is showing / no file gets displayed for apps installed through Trollstore. In my case it is happening iTorrent (https://onejailbreak.com/blog/itorrent/) which didn't have that issue with AltStore. 

Is it a known limitation?

**Comment by opa334** [comment]
Yes, you have to temporarely switch the app to a user registration for this to work

**Comment by jayavanth18** [comment]
AS of now i am still facing this issue.
